src/Handler/ArchiveHandler.php

```diff
@@ -45,16 +47,20 @@ protected function getChecksumData(): array
         return ['ignore' => $ignore] + parent::getChecksumData();
     }
 
-    protected function download(Composer $composer, IOInterface $io): void
+    public function install(Composer $composer, IOInterface $io): void
```

Comment:
won't changing this potentially break other things that use `download` and sub-class this?